packages/python-packages/low-level-client-api-gen/llc_api/llc_view_models.py

```diff
@@ -0,0 +1,547 @@
+        self.Tokens.append(token)
+
+    def begin_group(self, group_name=""):
+        """Begin a new group in LLC view by shifting to right
+        """
+        self.indent += 1
+
+    def end_group(self):
+        """End current group by moving indent to left
+        """
+        if not self.indent:
+            raise ValueError("Invalid intendation")
+        self.indent -= 1
+
+    def add_operation(self,operation):
+        self.Operations.append(operation)
+
+    def add_operation_group(self,operation_group):
+        self.Operation_Group.append(operation_group)
+    
+    def to_token(self): 
+    #Create view 
+        #Overall Name
+        self.add_keyword(None,self.namespace,None)
+        self.add_space()
+        self.add_punctuation("{")
+        self.add_new_line(1)
+
+        #Name of client
+        self.add_whitespace(1)
+        self.add_keyword(None,self.Name,self.namespace)
+        self.add_punctuation("(")
+        self.add_stringliteral(None,self.endpoint_type,None)
+        self.add_space()
+        self.add_text(None,self.endpoint, None)
+
+        self.add_punctuation(",")
+        self.add_space()
+        self.add_stringliteral(None,self.credential_type,None)
+        self.add_space()
+        self.add_text(None,self.credential,None)
+ 
+        self.add_punctuation(")")
+        self.add_new_line(1)
+
+        #Set Navigation
+        navigation = Navigation(self.namespace, None)
+        navigation.set_tag(NavigationTag(Kind.type_package))
+
+        for operation in self.Operation_Group:
+            #Add children
+            child_nav = Navigation(operation.operation_group, self.namespace+operation.operation_group)
+            child_nav.set_tag(NavigationTag(Kind.type_class))
+            navigation.add_child(child_nav)
+            # navigation1 = Navigation(operation, self.namespace+operation.operation_group)
+            # navigation1.set_tag(NavigationTag(Kind.type_class))
+
+               #Set up operations and add to token
+            for op in operation.operations:
+                #Add operation comments
+                child_nav1 = Navigation(op.operation, self.namespace+op.operation)
+                child_nav1.set_tag(NavigationTag(Kind.type_method))
+                child_nav.add_child(child_nav1)
+
+            operation.to_token()
+            my_ops = operation.get_tokens()
+            for o in my_ops:
+                self.add_token(o)
+      
+     
+            # operation.to_token()
+            # my_ops = operation.get_tokens()
+            # for o in my_ops:
+            #     self.add_token(o)
+        self.add_new_line()
+        self.add_punctuation("}")
+
+        self.add_navigation(navigation)
+
+        return self.Tokens  
+
+    def to_json(self):
+        obj_dict={}
+        self.to_token()
+        for key in JSON_FIELDS:
+            if key in self.__dict__:
+                obj_dict[key] = self.__dict__[key]
+        for i in range(0,len(obj_dict["Tokens"])):
```

Comment:
if you do this then you won't have to keep indexing into the list in the below code, you can just use `i`

```suggestion
        for i in obj_dict["Tokens"]:
```